vercel.json

```diff
@@ -596,14 +596,16 @@
         { "source": "/resources/running_workspace", "destination": "/extensions/Editor/Running/Running_Workspace" },
         { "source": "/resources/Running_Project", "destination": "/extensions/Editor/Running/Running_Project" },
         { "source": "/resources/running_project", "destination": "/extensions/Editor/Running/Running_Project" },
+        { "source": "/resources/Assets_Introduction", "destination": "/extensions/Editor/Assets/Assets_Introduction" },
+        { "source": "/resources/assets_introduction", "destination": "/extensions/Editor/Assets/assets_introduction" },
         { "source": "/resources/Adding_Meshes", "destination": "/extensions/Editor/Assets/Adding_Meshes" },
         { "source": "/resources/adding_meshes", "destination": "/extensions/Editor/Assets/Adding_Meshes" },
         { "source": "/resources/Adding_Materials", "destination": "/extensions/Editor/Assets/Adding_Materials" },
         { "source": "/resources/adding_materials", "destination": "/extensions/Editor/Assets/Adding_Materials" },
         { "source": "/resources/Adding_Textures", "destination": "/extensions/Editor/Assets/Adding_Textures" },
         { "source": "/resources/adding_textures", "destination": "/extensions/Editor/Assets/Adding_Textures" },
-        { "source": "/resources/Adding_Sounds", "destination": "/extensions/Editor/Assets/Adding_Sounds" },

```

Comment:
These are URL rewrites for legacy support. Since this is a new page you don't have to rewrite from Adding_Files to Adding Files, but actually from the old URL to the new URL.

Reply:
Thanks @RaananW! I understand better now !!
So I rewrited Adding_Sounds to Adding_Files because this documentation has been moved here :)

Reply:
I have amended the commit, thanks :)